src/fragments/lib-v1/storage/existing-resources.mdx

```diff
@@ -6,7 +6,7 @@ amplify import storage
 
 For more details, see how to [Use an existing S3 bucket or DynamoDB table](/gen1/[platform]/build-a-backend/storage/import/).
 
-If you are not using the Amplify CLI, an existing Amazon S3 bucket can be used by referencing it in your `amplifyconfiguration.json` file.
+If you are not using the Amplify CLI, an existing Amazon S3 bucket can be used by referencing it in your `amplifyconfiguration` file.
```

Comment:
*nit:* With the extension removed it probably makes sense to say "Amplify configuration file" rather than use the file name without an extension.

```suggestion
If you are not using the Amplify CLI, an existing Amazon S3 bucket can be used by referencing it in your Amplify configuration file.
```